yarn.lock

```diff
@@ -405,6 +405,15 @@
     "@babel/traverse" "^7.9.0"
     "@babel/types" "^7.9.0"
 
+"@babel/highlight@^7.10.4":
```

Comment:
Is this file change required @darshanhande11?

Reply:
@codesankalp These changes occurred in yarn.lock file after I ran `yarn`. So, I didn't do those changes voluntarily.

Reply:
Please unstage this file as it is not required here.

Reply:
@codesankalp I have removed the unrequired code from yarn.lock